daffodil-cli/src/test/resources/org/apache/daffodil/CLI/trace_input.dfdl.xsd

```diff
@@ -1,56 +1,56 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I made no changes to this. May just be that it was converted CRLF -> LF by my IDE? Or tabs to spaces?

Reply:
Yep, I think CRLF -> LF, same as the other file. Change is fine